### DISCUSSION TITLE
Installation of OpenEMS on Devuan

### DISCUSSION THREAD

**Discussion by HermanUAE** [thliebig/openEMS-Project#185] [General]
Hi everyone, OpenEMS is packaged for Devuan so I simply installed it from the repos, hoping that it would work - silly me, I know... :)  That also pulled in Octave, but there is some setup missing in Octave.  The documentation here _https://docs.openems.de/octave/install.html_ refers to paths like _/usr/share/openEMS/matlab_ and _usr/share/CSXCAD/matlab_, but I cannot find these paths.  Do I have to create these directories?  I'll try that, but if someone already figured this out, please help! Cheers, H.

**Comment by HermanUAE** [comment]
Good heavens.  I think I got it to work on the older Devuan: https://www.aeronetworks.ca/2024/02/openems-with-octave-and-scilab.html

**Comment by thliebig** [comment]
No the packages should have been created by the repos. You need to check where the matlab files have been installed and thenn tell Octave to find and use these... 
Alternatively you could install from sources?

**Comment by HermanUAE** [comment]
My recommendation to everyone is to install from source, since that is the only way to be sure that everything is built and installed right, but you may need an older version of your favourite Linux distro for the libraries to resolve, not the very latest. Installing openEMS on a virtual machine is recommended so that you don't accidentally mess up your host system.

Anyhoo, this looks like a useful tool.  Thank you very much!
